mythplugins/mythmusic/mythmusic/visualize.cpp

```diff
@@ -583,15 +590,17 @@ void WaveForm::saveload(MusicMetadata *meta)
     m_maxr = 0;
     m_sqrr = 0;
     m_position = 0;
-    m_lastx = WF_WIDTH;
+    m_lastx = m_wfsize.width();
     m_font = QApplication::font();
-// m_font.setPointSize(14);
-    m_font.setPixelSize(20);    // small to be mostly unnoticed
+    // m_font.setPointSize(14);
+    m_font.setPixelSize(m_size.height() / 60); // small to be mostly unnoticed
 }
 
 unsigned long WaveForm::getDesiredSamples(void)
 {
-    return (unsigned long) WF_AUDIO_SIZE;  // Maximum we can be given
+    // could be an adjustable class member, but this hard code works well
+    // return (unsigned long) WF_AUDIO_SIZE;  // Maximum we can be given
+    return 4096;           // maximum samples per update, may get less
```

Comment:
Its better to have the readable name here than a magic number.  See https://clang.llvm.org/extra/clang-tidy/checks/readability/magic-numbers.html

Reply:
Fixed to WFAudioSize now defined in visualize.h